### DISCUSSION TITLE
Can we use dhikuzu to start shikuzu

### DISCUSSION THREAD

**Discussion by JOGIRAFFER** [RikkaApps/Shizuku#575] [Ideas]
Recently, while surfing the web, I found a magical piece of software that is said to be based on shikuzu, and its name is dhikuzu
![Screenshot_20240813_115300_com.huawei.android.launcher.png](https://github.com/user-attachments/assets/475d3380-3e49-49b8-8f2e-b7bccc31a121)

This is also a github open source project,Its address is []()https://github.com/iamr0s/Dhizuku

The internal interface is as follows:
![Screenshot_20240813_115707_com.rosan.dhizuku.jpg](https://github.com/user-attachments/assets/1efd341b-dd85-4139-8e17-7c12d2361e9f)

It uses DO permissions, which are slightly higher than DA permissions, and can be unlocked continuously, and will not be affected by restarts, so I wonder if it is possible to start shikuzu with slightly lower permissions directly with the help of Dhikuzu.

You know, the Dhikuzu permission is not available in most software, and some software needs to be injected with dax plug-ins when using it, which is very troublesome. On the other hand, Shikuzu is supported by most software, so I thought that launching Shikuzu with the help of DHIKUZU might increase the user experience.

I'm a Huawei phone user with a HarmonyOS system, no wireless connection, no root access, all I can do is occasionally connect to the computer to get adb permissions, or with the help of dhikuzu permissions for certain purposes.

So, can we add a new launch option in shikuzu, if I can, I would appreciate it.

Anyway, thank you very much.

**Comment by apkshizuku-dev** [comment]
please confirm from offical site [https://shizukuapk.net](https://shizukuapk.net)
